plugins/modules/template_intent.py

```diff
@@ -265,282 +525,575 @@
+                            type: str
+                          range:
+                            description: Configuration Template Import Template's range.
+                            elements: dict
+                            suboptions:
+                              id:
+                                description: UUID of range.
+                                type: str
+                              maxValue:
+                                description: Max value of range.
+                                type: int
+                              minValue:
+                                description: Min value of range.
+                                type: int
+                            type: list
+                          required:
+                            description: Is param required.
+                            type: bool
+                          selection:
+                            description: Configuration Template Import Template's selection.
+                            suboptions:
+                              defaultSelectedValues:
+                                description: Default selection values.
+                                elements: str
+                                type: list
+                              id:
+                                description: UUID of selection.
+                                type: str
+                              selectionType:
+                                description: Type of selection(SINGLE_SELECT or MULTI_SELECT).
+                                type: str
+                              selectionValues:
+                                description: Selection values.
+                                type: dict
+                            type: dict
+                        type: list
+                      tags:
+                        description: Configuration Template Import Template's tags.
+                        elements: dict
+                        suboptions:
+                          id:
+                            description: UUID of tag.
+                            type: str
+                          name:
+                            description: Name of tag.
+                            type: str
+                        type: list
+                      template_content:
+                        description: Template content.
+                        type: str
+                      templateParams:
+                        description: Configuration Template Import Template's templateParams.
+                        elements: dict
+                        suboptions:
+                          binding:
+                            description: Bind to source.
+                            type: str
+                          customOrder:
+                            description: CustomOrder of template param.
+                            type: int
+                          dataType:
```

Comment:
are we still using camelcase?

Reply:
Export and import template and projects are blocked so we are using camel case similar to one to one API.